aries-site/src/data/structure.js

```diff
@@ -146,6 +146,7 @@ export const structure = [
     seoDescription:
       'HPE Design System dashboard template for screens featuring content within tiles.',
     sections: [],
+    relatedContent: ['Cards', 'Lists', 'Grid'],
```

Comment:
I wonder if we want to drive relatedContent via tags such as 'Inputs', 'Controls', 'Layout', etc. Then, we'd get the related content cards based on other pages with the same tags. This might be more efficient for cross linking between pages. We could even have it support both specific page names (in case we wanted to hand pick a page to show up in related) as well as the tags. Thoughts?

I'd say it could be an optimization to this set up, but I think for the first pass we could move forward like this.

Reply:
I'm open to approach like this. I was thinking flexibility to allow for use-case patterns emerge and then address as those patterns appear.